### ISSUE TITLE
[JENKINS-63193] Increase contrast of the disabled plugin uninstall button 

### ISSUE THREAD

**Issue by timja-bot** [timja-org/jenkins-gh-issues-poc-11-04#1495]
<p>When the "Uninstall" button on the /<em>pluginManager/installed</em> page is disabled it has too low contrast that can bring in accessibility issues. See screenshot.</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/51993/51993_Captura+de+pantalla+2020-07-24+a+las+10.50.18.png" height="282" width="598" style="border: 0px solid black" /></span></p>

<p>Solution:</p>

<ul>
	<li>Remove the CSS rules for the `#plugins tr.has-dependents input.uninstall` selector.</li>
	<li>Bump the opacity of the disabled button from 0.2 to 0.4 or 0.5.</li>
</ul>

---
<details><summary><i>Originally reported by <a title="Félix Queiruga Balado" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=fqueiruga">fqueiruga</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-63193" target="_blank">Increase contrast of the disabled plugin uninstall button </a></i></summary>
<i><ul>
<li><b>assignee</b>: <a title="Benny Cheng" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=benebsiny">benebsiny</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Minor
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2020-09-28T11:58:33+00:00
<li><b>imported</b>: 2025-11-04
</ul></i>
</details>

**Comment by timja-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=benebsiny">benebsiny</a>:</i>
<p>I've create a pull requests <a href="https://github.com/jenkinsci/jenkins/pull/4917" class="external-link" target="_blank" rel="nofollow noopener">here</a></p>

**Comment by timja-bot** [comment]
<i>[Epic: <a href="https://github.com/timja-org/jenkins-gh-issues-poc-11-04/issues?q=in%3Atitle%20JENKINS-60919%20label%3Aepic">JENKINS-60919</a>]</i>
